### ISSUE TITLE
Filtering ATS queryable links test not enforceable

### ISSUE THREAD

**Issue by aaime** [opengeospatial/ogcapi-features#962]
From Features Part 1:

> Links included in payload of responses SHOULD also be included as Link headers in the HTTP response according to [RFC 8288, Clause 3](https://docs.ogc.org/is/17-069r4/17-069r4.html#rfc8288). This recommendation does not apply, if there are a large number of links included in a response or a link is not known when the HTTP headers of the response are created.

And from Part 3, Filtering, when [talking about queryables](https://docs.ogc.org/is/19-079r2/19-079r2.html#queryables):

> > The Queryables resource SHALL be referenced from each Collection resource with a link with the link relation type http://www.opengis.net/def/rel/ogc/1.0/queryables (or, alternatively, [ogc-rel:queryables]).

And also:

> It is [recommended to include the link in the response via an HTTP header](http://www.opengis.net/doc/IS/ogcapi-features-1/1.0#_link_headers).

However the ATS in [A.1.2](https://docs.ogc.org/is/19-079r2/19-079r2.html#_conformance_test_2) assumes that links can be found in the response header. It should check the response contents instead, where the presence of the link can be mandated.

Specific point in case, GeoServer has many output supported output formats, leading to the "too many links to fit in the header" case. Regardless, besides the specific reason why the queryable link is not present in the header, a test should not assume recommended behavior as mandatory.

**Comment by cportele** [comment]
Good point. If the response does not include any Link header, the test should not fail.
